cluster/src/main/scala/io/snappydata/ToolsCallbackImpl.scala

```diff
@@ -327,4 +327,13 @@ object ToolsCallbackImpl extends ToolsCallback with Logging {
     val prop = taskProps.getProperty(Constant.REPL_OUTPUT_DIR)
     SnappyInterpreterExecute.getLoader(prop)
   }
+
+  override def getScalaCodeDF(code: String,
+    session: SnappySession, options: Map[String, String]): Dataset[Row] = {
+    SnappyInterpreterExecute.getScalaCodeDF(code, session, options)
+  }
+
+  override def closeAndClearScalaInterpreter(uniqueId: Long) = {
```

Comment:
Idea shows warning as return type is ommitted for the function. Need to add return type (Unit)